hack/run-wmcb-ci-e2e-test.sh

```diff
@@ -39,6 +39,12 @@ if [ -z "$ARTIFACT_DIR" ]; then
   export ARTIFACT_DIR=$ARTIFACT_DIR
 fi
 
+DOCKER_RUNTIME=${DOCKER_RUNTIME:-}
+if [ -z "$DOCKER_RUNTIME" ]; then
+  echo "DOCKER_RUNTIME is not set. Defaulting to true"
```

Comment:
The `--dockerRuntime` flag option defaults to true [already](https://github.com/openshift/windows-machine-config-bootstrapper/pull/322/commits/de59d0c49effd43b67dd8a90223fb281715463ee#diff-933b7d0fd790e2d3de8ea9dc1185e3441f8ea6aea0adac993cadb0567b9296fbR65) if is not provided.

Ideally, if `DOCKER_RUNTIME` is provided you should set the `-dockerRuntime=true` arg in  `internal/test/wmcb/deploy/job.yaml`.

Reply:
yes, DOCKER_RUNTIME value is appended in internal/test/wmcb/deploy/job.yaml, this was just a safe check to avoid appending null value if the var is not set. Modified it to removing the line if the value is not set.